src/zui/ZUINumberChip/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react';
+import { makeStyles } from '@mui/styles';
+import { Box, Theme } from '@mui/material';
+
+type ZUINumberChipProps = {
+  color: string;
+  size?: keyof typeof FONT_SIZES;
```

Comment:
is there an easys way to explain how this works, bc i do not understand this type...

Reply:
`FONT_SIZES` is an object. When I do `typeof FONT_SIZES` I get the type of that object (which in this is identical to the object value because the object is declared `as const` so it will always look the same).

The `keyof` keywords takes an map/record/object type and gives you a union of it's keys. 

This means that this expression (`keyof typeof FONT_SIZES`) evaluates to the type `'lg' | 'md' | 'sm'`.

Reply:
thank you! :D